mule-migration-tool-expression/src/main/scala/com/mulesoft/tools/MelGrammar.scala

```diff
@@ -256,6 +261,14 @@ class MelGrammar(val input: ParserInput) extends Parser with StringBuilding {
     ws ~ "contains"
   }
 
+  def dwScript = rule {
+    ws ~ str("\"") ~ stringPreserveEscaping ~ str("\"") ~> createStringNode
+  }
+
+  def dwMethodInvocation: Rule1[DWFunctionNode] = rule {
+    ws ~ str("dw") ~ ws ~ ch('(') ~ ws ~ dwScript ~ ws ~ ch(')') ~> createDWMethodInvocationNode
```

Comment:
Why do we need to have `dw` in the grammar and is not just a method invocation

Reply:
makes sense, I updated it in the new rev